templates/gogo/file.go

```diff
@@ -46,6 +46,12 @@ var (
 // define the regex for a UUID once up-front
 var _{{ snakeCase .File.InputPath.BaseName }}_uuidPattern = regexp.MustCompile("^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$")
 
+// define the regex for an HTTP header name once up-front
+var _{{ snakeCase .File.InputPath.BaseName }}_httpHeaderName = regexp.MustCompile("^:?[0-9a-zA-Z!#$%&'*+-.^_|~\x2C]+$")
```

Comment:
Hmm, is there some way we can have the same Go file included here and above to ensure single source-of-truth for regexes?

Reply:
Oops, this was an artifact from before. No longer needed, the well_known_regex is converted to a regex pattern in the checker module, so the later code doesn't need the definition anymore.